frontend/src/app/forms/forms.page.ts

```diff
@@ -132,23 +126,29 @@ export class FormsPage implements OnInit, OnDestroy {
   }
 
   /*
-  @Usage In this method navigation to home page is handled.
-  */
-  onClose(): void {
+   * In this method navigation to home page is handled.
+   */
+  closeForm(): void {
     this.navigationController.navigateForward(['/home']);
   }
 
   /*
-  @Usage In this method navigation to respective step is handled.
-  */
-  onFormStepsNavigation(event: {
+   * In this method navigation to respective step is handled.
+   * @param {Object} menu - Contains the menu to navigate
+   * @param {number} menu.id - The unique id for the menu. Used in query params.
+   * @param {string} menu.displayName - The name to be displayed.
+   * @param {boolean} menu.isCompleted - The completion status of a step menu.
+   * @param {boolean} menu.isActive - The active status of a step menu.
+   * @param {string} menu.selector - The selector name of the step menu component.
+   */
+  navigateToStep(menu: {
```

Comment:
When you use the comments that start like `/**` instead of `/*`, then you will get some error messages by TSLint. My suggestion: First add a new interface into app/forms/model/menu-item.model.ts:

``` TS
/**
 * Some explanation maybe... 
 */
export interface MenuItem {
  /** The unique id for the menu. Used in query params. */
  id: number;
  /** The name to be displayed */
  displayName: string;
  /** The completion status of a step menu. */
  isCompleted: boolean;
  /** The active status of a step menu. */
  isActive: boolean;
  /** The selector name of the step menu component. */
  selector: string;
}
```

Then, change your method like this:
``` TS
/**
 * In this method navigation to respective step is handled.
 * @param menu Contains the menu to navigate
 */
navigateToStep(menu: MenuItem)
```

Reply:
(There is no need in TypeScript to write `@param {Object} menu`. Just write `@param menu`. TS knows the type anyways.)

Reply:
I am currently working structuring interfaces including basicInformation interfaces. So together with interface changes this will be done. I already have it in my list.

Reply:
Now just modifies @param menu. Later on menu will be written as interface.Hence resolving it.